### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by KarthikRaju391**
### Following features have been added

- the logo is a SVG
- the logo links to the home page
- the text on the left links to https://profy.dev/employers
- the text on the right links to /terms

### CODE REVIEW

**Comment by jkettmann** [comment]
You wrote a comment in another PR, but let's start here since this task has to be done first. You can see that the end-to-end tests are failing

image

The test that's failing is this.

image

This is the last acceptance criteria in the Clickup task for the header

image

The source code of the test that's failing is [here](https://github.com/profydev/reddit-timer-KarthikRaju391/blob/main/cypress/integration/2-Header.spec.js#L20)

image

TL;DR

You just have to add `javascript` to the URL. Your search page URL is currently `/searchpage`. Replace it with something like `/search/javascript`

**Comment by KarthikRaju391** [comment]
Ohk thank you so much!

**Comment by mentor-tara** [comment]
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime, consider starting with the next task already to practice the professional Git workflow. [If you need a refresher, feel free to follow this section in the course again.](https://profy.dev/project/reddit-timer/while-waiting)[](ckmadhrlk3ng80b35a5qccwi8)

**Review by mentor-tara — CHANGES_REQUESTED** [review]
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] Use human readable URLs [:link:](https://profy.dev/project/reddit-timer/review-tips/human-readable-links)
- [x] Use absolute instead of relative links: `<Link to="/search" />` [:link:](https://profy.dev/project/reddit-timer/review-tips/use-absolute-links)
- [x] No components that solely copy props to their children [:link:](https://profy.dev/project/reddit-timer/review-tips/unnecessary-components)
- [x] Optimize svg files with SVGOMG [:link:](https://profy.dev/project/reddit-timer/review-tips/optimize-svg-with-svgomg)
- [x] Use react-router Link instead of HTML anchor tag [:link:](https://profy.dev/project/reddit-timer/review-tips/link-instead-of-anchor-tag)

**Styles:**
- [x] **Share styles between internal and external links** [:link:](https://profy.dev/project/reddit-timer/review-tips/shared-styles-between-internal-and-external-links)
- [x] Don't use `justify-content: space-between` here [:link:](https://profy.dev/project/reddit-timer/review-tips/no-space-between-in-footer)
- [x] **Extract shared CSS values** [:link:](https://profy.dev/project/reddit-timer/review-tips/common-css-values-in-theme)
- [x] Be careful with pixel-perfectness when using relative font sizes like `0.9em` [:link:](https://profy.dev/project/reddit-timer/review-tips/careful-with-relative-font-size)
- [x] Don't use 250px padding for the footer wrapper [:link:](https://profy.dev/project/reddit-timer/review-tips/no-250px-footer-padding)
- [x] **Refrain from styling children with CSS selectors** [:link:](https://profy.dev/project/reddit-timer/review-tips/css-selectors)

**Tests:**
- [x] Get the logo by role with `getByRole` [:link:](https://profy.dev/project/reddit-timer/review-tips/get-logo-by-role)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/footer-713055ad-9fe5-41b2-87c0-9a434095cc14). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckmaax69k3m8h0b35eyw85myf)

**Review by mentor-tara — APPROVED** [review]
Great work! :tada::tada::tada: You can merge the PR now.[](ckmadmft43nqo0a84r1nmtdyr)
